cmd/kubelet/app/server.go

```diff
@@ -474,6 +476,30 @@ func makeEventRecorder(kubeDeps *kubelet.Dependencies, nodeName types.NodeName)
 	}
 }
 
+func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) cpuset.CPUSet {
+	emptyCPUSet := cpuset.NewCPUSet()
+	if cpus != "" {
+		topo, err := topology.Discover(machineInfo)
+		if err != nil {
+			klog.Error("Unable to discover CPU topology info")
+			return emptyCPUSet
+		}
+		reservedCPUSet, err := cpuset.Parse(cpus)
+		if err != nil {
+			klog.Error("Unable to parse reserved-cpus list")
+			return emptyCPUSet
```

Comment:
Will this silently ignore the given cpuset if it isn't possible to parse it? Are we sure we don't want to fail hard if that it the case? This will probably be caught by config validation, or?

Reply:
sure